TestApps/MessagingDemoAppSwiftUI/AppPages/CardsView.swift

```diff
@@ -129,6 +152,22 @@ class CardCustomizer : ContentCardCustomizing {
         template.dismissButton?.image.iconFont = .system(size: 10)
     }
     
+    func customize(template: ImageOnlyTemplate) {
```

Comment:
if we switch to generic customization, implementation would look something like 
```
func customize<T>(template: T) where T : ContentCardTemplate {
    switch template: {
        case is ImageOnlyTemplate:
            // customize image only template
        etc...
    }
}
```

the code isn't necessarily cleaner from the app developer's perspective, but i'm a big fan of having only a single protocol method in `ContentCardCustomizing`.  

thoughts?